classes/profile_table.php

```diff
@@ -146,4 +155,25 @@ public function col_user(object $record): string {
             }
         }
     }
+
+    /**
+     * Display for the delete column
+     *
+     * @param object $record
+     * @return mixed
+     * @throws \coding_exception
+     * @throws \moodle_exception
+     */
+    public function col_delete(object $record) {
+        global $OUTPUT;
+        $deleteurl = new \moodle_url('/admin/tool/excimer/delete.php', [ 'deleteid' => $record->id, 'sesskey' => sesskey() ]);
+        $link = new \action_link(
+                $deleteurl,
+                '',
+                new \confirm_action('Confirm?'),
+                null,
+                new \pix_icon('t/delete', get_string('deleteprofile', 'tool_excimer'))
+        );
```

Comment:
Perhaps instead of having this split into multiple lines such as this, it might be clearer to have something similar to this:
```
        $confirmaction = new \confirm_action(/* fix with lang string */);
        $deleteicon = new \pix_icon('t/delete', get_string('deleteprofile', 'tool_excimer'));
        $link = new \action_link($deleteurl, '', $confirmaction, null,  $deleteicon);
```